src/Symfony/Component/Console/Completion/CompletionInput.php

```diff
@@ -53,7 +53,7 @@ public static function fromString(string $inputStr, int $currentIndex): self
      * Create an input based on an COMP_WORDS token list.
      *
      * @param string[] $tokens       the set of split tokens (e.g. COMP_WORDS or argv)
-     * @param          $currentIndex the index of the cursor (e.g. COMP_CWORD)
+     * @param int      $currentIndex the index of the cursor (e.g. COMP_CWORD)
```

Comment:
This is intended.

I think we should rather remove all types everywhere if they duplicate the type defined in PHP (which is almost always the case, except for generic types). See https://github.com/symfony/symfony/pull/53363#discussion_r1439867370

But if we do this, we should automate this using Rector or PHP CS Fixer.

Reply:
This is an interesting point of view. Are we 100% we want to do this? Rector could definitely do the trick with this

Reply:
Yes we are :)

Reply:
Alright, I might close this PR in favor of one that removes redundant types as explained by Wouter. Should such PR target 5.4 or 7.1?